wrappers/tokio/README.md

```diff
@@ -0,0 +1,34 @@
+# ShuttleTokio
+
+This folder contains the implementation and wrapper for ShuttleTokio, which provides [tokio](https://crates.io/crates/tokio) support to Shuttle.
+
+## How to use
+
+To use it, do something akin to the following in your Cargo.toml:
+
+```
+[features]
+shuttle = [
+   "tokio/shuttle",
+]
+
+[dependencies]
+tokio = { package = "shuttle-tokio", version = "VERSION_NUMBER" }
+```
+
+The code will then behave as before when the `shuttle` feature flag is not provided, and will run with Shuttle-compatible primitives when the `shuttle` feature flag is provided.
+
+## Limitations
+
+The development of ShuttleTokio has come as a consequence of teams at Amazon needing support for model checking code using tokio. What has driven development is the functionality needed by these teams, meaning there are parts of tokio which have not been implemented or which have not been modeled faithfully. This is something to keep in mind if you want to use ShuttleTokio, and there is some likelihood that there will be functionality that will need to be added. We are happy to review PRs if there is functionality you'd like to add to ShuttleTokio.
+
+A non-exhaustive look of current limitations:
```

Comment:
Not sure we should put this list in the code.  Let's create an issue for it.  (One issue is fine, don't need to create separate issues for each item.)